### PR TITLE
feat: cannot edit a wish with votes

### PR DESCRIPTION
**Pull request by pixelass**
## Motivation
Warn about editing on voted wishes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "@apollo/react-hooks": "4.0.0",
     "@contentful/rich-text-react-renderer": "15.1.0",
     "@contentful/rich-text-types": "15.1.0",
-    "@dekk-app/dekk-backend": "1.5.3",
+    "@dekk-app/dekk-backend": "1.6.0",
```
Comment (TimPietrusky) [inline package.json]: Please use 1.6.1

**Review by TimPietrusky — CHANGES_REQUESTED** [review]
@pixelass I have one change request!

File: package.json (hunk 1 above)
Comment (pixelass) [reply package.json]: thx

**Review by TimPietrusky — APPROVED** [review]
LGTM ❤️

**Pull request merged**